packages/office-ui-fabric-react/src/components/Panel/examples/Panel.Medium.Example.tsx

```diff
@@ -15,7 +15,13 @@ export class PanelMediumExample extends React.Component<{}, IPanelMediumExampleS
     return (
       <div>
         <DefaultButton secondaryText="Opens the Sample Panel" onClick={this._showPanel} text="Open Panel" />
-        <Panel isOpen={this.state.showPanel} onDismiss={this._hidePanel} type={PanelType.medium} headerText="Medium Panel">
+        <Panel
+          closeButtonAriaLabel="Close"
+          isOpen={this.state.showPanel}
+          onDismiss={this._hidePanel}
+          type={PanelType.medium}
+          headerText="Medium Panel"
```

Comment:
Minor note but language guidelines I've always referred to encourage Sentence casing rather than Title Casing for pretty much all strings.

Reply:
Sounds good. Wasn't intentionally modifying this string, just happened to pop up because prettier re-wrapped these lines. Will address.